File: Week 6 Python/Problem Set 6/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int

# Get the height of the mario pyramid from user
height = get_int("Height: ")

# Prompt the user forever until valid height number
while height < 1 or height > 8:
    height = get_int("Height: ")

# For each row in the pyramid
for i in range(height):
    # For each column in the pyramid
    for j in range(height-1, -1, -1):
        # Print space
        if j > i:
            print(" ", end="")
        # Print blocks
        else:
            print("#", end="")
    # Print next line
    print()

File: Week 1 C/Problem Set 1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // declare integer
    int height;

    // Prompt the user to get the height of the mario pyramid
    do
    {
        height = get_int("Height: ");
    } // Prompt untill valid input
    while (height < 1 || height > 8);

    // Build rows in accordance with the height of the pyramid
    for (int i = 0; i < height; i++)
    {
        // Print # for each row

        for (int j = height - 1; j >= 0; --j)
        {
            // printf("#");
            if (j <= i)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }
        printf("\n");
    }
}
